index.html

```diff
@@ -3,21 +3,25 @@
 
 <head>
     <meta charset="UTF-8" />
-    <title>sandbox</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="style.css" />
     <link
         href="https://fonts.googleapis.com/css?family=Roboto+Mono:400,100,300italic,300,100italic,400italic,500,500italic,700,700italic&;subset=latin,cyrillic"
         rel="stylesheet">
+    <title>sandbox</title>
 </head>
 
 <body>
     <div class="container">
         <div class="cover">
-            <div class="name"> Michael Lindsay </div>
+            <div class="name"> MICHAEL D LINDSAY </div>
```

Comment:
I KNOW my boy ain't using `div`s when a more semantic alternative is available.
```suggestion
            <h1 class="name">MICHAEL D LINDSAY</h1>
```